CorsixTH/Lua/app.lua

```diff
@@ -671,27 +691,21 @@ function App:readLevelFile(level)
   return level_info
```

Comment:
Not your fault probably, but if you look at this code you'll notice that 662 to 665 and 667 to 670 do the exact same thing, except for different pieces of text.

By encapsulating the 4 lines into a function and giving it `contents` and `"Briefing"` resp `"Debriefing"` you can move the 10 lines of details out of this function (for as far as I can see now), and replace it by 2 calls to the new function.

By doing that more, you get higher-level functions that handle bigger steps in the code without getting several hundreds of lines long. Reading and understanding such functions is much easier too, as you never even see all the little details that are hidden in the called functions.